### DISCUSSION TITLE
List collection in Modrinth API?

### DISCUSSION THREAD

**Discussion by aayushdutt** [modrinth/.github#233] [Feature requests]
How do I list the projects under a user collection using Modrinth API?
Couldn't find it in the [API docs](https://docs.modrinth.com/)

**Comment by Prospector** [comment]
Collections are currently a part of the WIP undocumented v3 API. When we consider the API stable, docs for it will be published.

**Reply by aayushdutt** [reply]
Thanks that sounds good.
Took a quick look at the [collections route](https://github.com/modrinth/labrinth/blob/master/src/routes/v3/collections.rs) and I was able to access it from `https://api.modrinth.com/v3/collection/<collection_id>`

**Reply by Prospector** [reply]
Nice, just make sure anything you use this for has a good amount of error handling in case the route changes/breaks before it stabilizes (unlikely but good to account for it)
